File: class/class.py
class Person:
    def __init__(self,name,age):
        self.name = name
        self.age = age
        self.gomi = "mogeogfsdjakl"

    def __del__(self):
        print("clash")
    def show(self) -> str:
        print("{0}さん{1}歳".format(self.name,self.age))
        return "str"

class BussinesPerson(Person):
    def __init__(self, name, age,Bussines):
        super().__init__(name,age)
        self.Bussines = Bussines
    
    def __del__(self):
        return super().__del__()
    
    def show(self) -> str:
        print("{0}さん{1}歳　職業 {2}".format(self.name,self.age,self.Bussines))
        return "str"

p = Person("スズキ",23)
p.show()
print(p.age)
print(p.gomi)
bp = BussinesPerson("田中",22,"無職")

bp.show()

File: class/class.js
class Person {
    constructor(name,age) { 
        this.name = name;
        this.age = age;
    }
    show() { 
        console.log(this.name+"さん "+this.age+"歳")
    }
}

class MathPerson extends Person {
    constructor(name,age,math) {
        super(name,age);
        this.math = math
    }
    show_math() { 
        console.log(this.name +"さん" + this.math + "を学んでいます");
    }
}

let p = new Person("スズキ",32);
p.show();

let mp = new MathPerson("田中",53,"代数");
mp.show();
mp.show_math();